packages/app/src/cli/models/extensions/specifications/app_config_webhook_subscription.ts

```diff
@@ -50,14 +50,15 @@ function transformFromWebhookSubscriptionConfig(content: object) {
   const {api_version, subscriptions = []} = webhooks
 
   const webhookSubscriptions = subscriptions.flatMap((subscription) => {
-    const {uri, topics, ...optionalFields} = subscription
+    // compliance_topics gets handled by privacy_compliance_webhooks
+    const {uri, topics, compliance_topics: _, ...optionalFields} = subscription
```

Comment:
needed `compliance_topics: _` to prevent `compliance_topics` to get pushed as part of `subscription` in the `webhook_subscription` config, which gives us errors